### PR TITLE
[RAPTOR-9177] Upgrade dr-usertool to 2.x

### PR DESCRIPTION
**Pull request by rvorobii**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

### CODE REVIEW

**Comment by rvorobii** [comment]
jenkins run test_integration_general

**Comment by rvorobii** [comment]
jenkins run test_integration_python_keras

**Comment by rvorobii** [comment]
jenkins run test_integration_r_environment

**Comment by rvorobii** [comment]
jenkins run continuous-integration/jenkins/pr-merge

**Comment by rvorobii** [comment]
jenkins run test_integration_general

File: tests/functional/conftest.py [hunk 1]
```diff
@@ -33,24 +40,23 @@ def pytest_configure(config):
...
     if not config.pluginmanager.getplugin("dsession"):
         suffix = str(uuid.uuid4().int)
-        env, db = dr_usertool_setup()
+        env = dr_usertool_setup()
+        admin_api_key = get_admin_api_key()
 
         # User credentials
         user_username = "local-custom-model-templates-tests-{}@datarobot.com".format(suffix)
-        user_password = "lkjkljnm988989jkr5645tv_{}".format(suffix)
+        user_password = "Lkjkljnm988989jkr5645tv_{}".format(suffix)
```
Comment (rvorobii) [inline tests/functional/conftest.py:48]: an uppercase character is required

Comment (burkestar) [reply tests/functional/conftest.py:48]: isn't it nice that business logic rules now are enforced for user creation ;)

File: jenkins/test_drop_in_envs.groovy [hunk 2]
```diff
@@ -22,6 +22,8 @@ node('release-dev && memory-intense'){
         popd
     '''.stripIndent()
 
+    createInitialAdminUser()
```
Comment (burkestar) [inline jenkins/test_drop_in_envs.groovy:25]: Maybe leave a comment that explains which env var gets exported (`APP_ADMIN_API_KEY`) and a hint about where this groovy function comes from.  That way there are no mysteries for our future selves.

**Review by burkestar — APPROVED** [review]

File: jenkins/test_drop_in_envs.groovy (hunk 2 above)
Comment (zohar-mizrahi) [reply jenkins/test_drop_in_envs.groovy:25]: Yeah, good point. 
Where does it come from?

**Review by zohar-mizrahi — APPROVED** [review]

**Pull request merged**